sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -677,13 +678,15 @@ public String getId() {
             final BatchExecutor executor = new BatchExecutor(this, transactionalBatch, transactionalBatchRequestOptions);
             final Mono<TransactionalBatchResponse> responseMono = executor.executeAsync();
 
-            return database.getClient().getTracerProvider().
-                traceEnabledBatchResponsePublisher(
-                    responseMono,
-                    context,
-                    this.batchSpanName,
-                    database.getId(),
-                    database.getClient().getServiceEndpoint());
+            return database.getClient().getTracerProvider().traceEnabledBatchResponsePublisher(responseMono,
```

Comment:
@moderakh please share the style guidance.

Reply:
@deparash please try to change
```suggestion
            return database.getClient()
                           .getTracerProvider()
                           .traceEnabledBatchResponsePublisher(responseMono,
                               context,
                               this.batchSpanName,
                               this.getId(),
                               database.getId(),
                               database.getClient(),
                               transactionalBatchRequestOptions.getConsistencyLevel(),
                               OperationType.Batch,
                               ResourceType.Document);
        });
```

in general please avoid having all the method calls in one line for better readability.